framework/source/class/qx/dev/unit/Sinon.js

```diff
@@ -74,8 +70,10 @@ qx.Bootstrap.define("qx.dev.unit.Sinon",
 
 
 /**
- * @ignore(module, require, global, process.*, setImmediate)
+ * @ignore(module, require, global, process.*, setImmediate.*)
```

Comment:
`setImmediate` is a function, not an object with members. Why does it need `.*` here? Seems wrong.

Reply:
Done